release.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+RELEASE_LEVEL=$1
+
+git remote -v | grep "^upstream.*github\.com.jfly\/shtuff\.git" &> /dev/null
+
+if [[ $? -gt 0 ]]; then
```

Comment:
I think these 2 lines could get combined:

```bash
if ! git remote -v | grep "^upstream.*github\.com.jfly\/shtuff\.git" &> /dev/null; then
```

(same for your other if statements in this file).